pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -164,13 +172,47 @@ func (ic *ifConfigurator) createContainerLink(endpointName string, result *curre
 	if err != nil {
 		return nil, err
 	}
+	containerIPStr, err := parseContainerIPs(result.IPs)
+	if err != nil {
+		klog.Errorf("Failed to find container %s IP", containerID)
+	}
+	// Save interface config to HNSEndpoint. It's used for creating missing OVS
+	// ports during antrea-agent boot stage. The change is introduced mainly for
+	// ContainerD support. When working with ContainerD runtime, antrea-agent creates
+	// OVS ports in asynchronously way. So the OVS ports will be lost if antrea-agent
+	// get stopped/restarted before port creation completes.
+	//
+	// The interface config will be rebuilt based on the params saved in the "AdditionalParams"
+	// field of HNSEndpoint.
+	//   - endpointName: the name of the host interface without Hyper-V prefix(vEthernet).
+	//     The name is same as the OVS port name and HNSEndpoint name.
+	//   - containerID: Used as key for goroutine lock to avoid concurrency issue.
+	//   - podName and PodNamespace: Used to identify the owner of the HNSEndpoint.
+	//   - dummyMac: the MAC address of the HNSEndpoint is unknown before we create it.
```

Comment:
Sorry, then why we need to save dummyMAC? To indicate the real MAC is not known?

Reply:
We save a dummyMAC to avoid printing error in parseConfig(). The parseConfig fuinction will print error if MAC address not exists.

Reply:
I would change ParseOVSPortInterfaceConfig() in this case, like a flag to indicate whether MAC should be present, or change to let its callers check whether MAC is present.

Reply:
Sounds good. I add a `checkMac` param to `ParseOVSPortInterfaceConfig()` to determine if print error msg after parse MAC.